tools/storage/tests/reap_worktrees_test.sh

```
@@ -0,0 +1,177 @@
+
+# assert_eq <case> <label> <actual> <expected>
+assert_eq() {
+  if [ "$3" = "$4" ]; then
+    echo "  PASS  $1: $2 = '$4'"; PASS=$((PASS+1))
+  else
+    echo "  FAIL  $1: $2 = '$3', expected '$4'"; FAIL=$((FAIL+1))
+  fi
+}
+
+run_plan() {  # run_plan <repo-dir> <cwd> [args...] -> writes plan to $PLAN
+  local repo="$1" where="$2"; shift 2
```

Shellcheck (0.11.0)

[warning] repo appears unused. Verify use (or export if used externally).

(SC2034)